flask-be/base.py

```diff
@@ -6,6 +6,10 @@
 
 app = Flask(__name__)
 
+# client id and secret needed for github oauth
+CLIENT_ID = "88ca09b0077cb16d7a39"
+CLIENT_SECRET = "3ff95633626be127d1513db0ba9a3aca56e9bebe"
```

Comment:
Definitely going to want to take this out and replace it with a new field in token.json .

Reply:
https://docs.github.com/en/apps/oauth-apps/building-oauth-apps/best-practices-for-creating-an-oauth-app#secure-your-apps-credentials

Reply:
That's true! I move it to the token.json file and will retrieve it dynamically while the server/client is running. But instead of letting everyone who runs the program generate their own pair of client id/secret, I believe we could use the same copy because it is the id and secret for the oAuth app I registered for the documentation app. I think we could keep the json file in a more secure place (such as google cloud?) instead of uploading it to our public repo.

Reply:
Yup, that makes sense to me. token.json is marked in the gitignore, so it won't get uploaded to our public repo. Maybe just toss the file in slack for the time being until we figure out what to do?

Reply:
Sure. I'll update the readme and send the file to slack.

Reply:
Thanks so much!